### ISSUE TITLE
Support redis？

### ISSUE THREAD

**Issue by ShawshankLin** [Qovery/Replibyte#179]

**Comment by benny-n** [comment]
As opposed to the other databases we support (PostgreSQL, MySQL, MongoDB) it seems that Redis has no official dump and restore tools (please correct me if I'm wrong here).

However, I did find this tool: https://github.com/delano/redis-dump
Which allows to dump the DB into a JSON file and then restore it from that JSON.
This tool is exactly what would be needed to support Redis.

The problem is that it looks like this tool isn't maintained at all (last commit ~4 years ago)

We could try to imitate the logic in this tool, but that would take some time.

WDYT @evoxmusic ?

**Comment by evoxmusic** [comment]
I would prefer to have more upvotes for Redis before supporting it. At the moment it seems to be an isolated request. WDYT @benny-n @fabriceclementz ?

**Comment by fabriceclementz** [comment]
I don't think we should concentrate our efforts on redis at the moment, in my use cases and to my knowledge, redis is generally used more as a cache layer, session store or as a queue, hence the interest of using replibyte seems less to me. Let the community show interest in Redis support and describe its use cases first.
